src/lib/server/memberProviders/d4hMembersProvider.ts

```diff
@@ -78,6 +78,18 @@ export default class D4HMembersProvider implements MemberProvider {
     }
   }
 
+  async getParticipantInfo(memberId: string) {
+    await this.initialize();
+    for (const token in this.tokenFetchInfo) {
+      const member = this.tokenFetchInfo[token].lookup[memberId].response;
+      if (!member) continue;
+      return {
+        email: member.email,
+        mobilephone: member.mobilephone,
+      } as MemberInfo;
```

Comment:
This should be `ParticipantInfo`? One thing I've done in the past is assign to a variable first, to make sure the type is correct, and then return:
```suggestion
      const result: ParticipantInfo {
        email: member.email,
        mobilephone: member.mobilephone,
      };
      return result;
```